internal/console/pre_scan.go

```diff
@@ -136,6 +139,18 @@ func (console *console) preScan() {
 	fmt.Println(versionMsg)
 	log.Info().Msgf(strings.ReplaceAll(versionMsg, "\n", ""))
 
+	log.Info().Msgf("Operating system: %s", runtime.GOOS)
+
+	mem, err := memory.Get()
+	if err != nil {
+		log.Info().Msgf("%s\n", err)
```

Comment:
Shouldn't be logged as error instead o f info, in this case?

Reply:
Yes, my bad. On it